composer.json

```diff
@@ -12,13 +12,18 @@
   "require": {
     "php": ">=5.6.0",
     "yiisoft/yii2": "^2.0.5",
-    "codeception/codeception": "^2.2",
+    "codeception/codeception": ">=2.2",
     "fillup/fake-bower-assets": "^2.0"
```

Comment:
This "fillup/fake-bower-assets" can cause problems for any code using this library. I don't know whether it's worth fixing that in this PR or a separate one, but it's a relatively easy fix:

1. Remove the `fillup/fake-bower-assets` line.
2. Update this composer.json file to use [asset-packagist](https://asset-packagist.org/) by adding this as a new root-level property in this composer.json file:

```
"repositories": [
    {
        "type": "composer",
        "url": "https://asset-packagist.org"
    }
]
```